<!DOCTYPE html>
<html style="font-size: 16px;" lang="en"><head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="keywords" content="Creative Portfolio, i am designer, webuilddigital, We design People Inspired Experiences that create positive change in people&amp;apos;s lives., Visual communication for brands and people">
    <meta name="description" content="">
    <title>Publications</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="Publications.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <meta name="generator" content="Nicepage 7.0.3, nicepage.com">
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,500,500i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "goncalojustino"
}</script>
    <meta name="theme-color" content="#1f203a">
    <meta property="og:title" content="Publications">
    <meta property="og:type" content="website">
  <meta data-intl-tel-input-cdn-path="intlTelInput/"></head>
  <body data-path-to-root="./" data-include-products="false" class="u-body u-xl-mode" data-lang="en"><header class="u-clearfix u-header u-header" id="sec-80dc"><div class="u-clearfix u-sheet u-sheet-1">
        <nav class="u-menu u-menu-one-level u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-top-bottom-menu-spacing u-hamburger-link u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg class="u-svg-link" viewBox="0 0 24 24"><use xlink:href="#menu-hamburger"></use></svg>
              <svg class="u-svg-content" version="1.1" id="menu-hamburger" viewBox="0 0 16 16" x="0px" y="0px" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg"><g><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</g></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="./" style="padding: 10px 20px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="About-me.html" style="padding: 10px 20px;">About me</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="Research.html" style="padding: 10px 20px;">Research</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="Team.html" style="padding: 10px 20px;">Team</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="Publications.html" style="padding: 10px 20px;">Publications</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-inner-container-layout u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="./">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-me.html">About me</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Research.html">Research</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Team.html">Team</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Publications.html">Publications</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
        <h2 class="u-subtitle u-text u-text-default u-text-1">Gonçalo C. Justino</h2>
      </div></header>
    <section class="u-align-center u-clearfix u-container-align-center u-section-1" id="carousel_a717">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <h2 class="u-text u-text-1">Publications</h2>
      </div>
    </section>
    <section class="u-clearfix u-container-align-center u-section-2" id="sec-b2e0">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="data-layout-selected u-clearfix u-expanded-width u-gutter-0 u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-align-center u-container-style u-layout-cell u-right-cell u-size-24 u-size-xs-60 u-layout-cell-1" src="">
                <div class="u-container-layout u-valign-middle u-container-layout-1" src="">
                  <img class="u-expanded-width u-image u-image-1" src="images/1-s2.0-S0010482524014781-gr4.jpg" data-image-width="633" data-image-height="553">
                </div>
              </div>
              <div class="u-align-left u-container-align-left u-container-style u-layout-cell u-left-cell u-size-36 u-size-xs-60 u-white u-layout-cell-2" src="">
                <div class="u-container-layout u-container-layout-2">
                  <h3 class="u-text u-text-1"> Comparison of two metabolomics platforms to discover biomarkers in critically ill patients from serum analysis</h3>
                  <p class="u-text u-text-2"><span class="u-icon"></span> &nbsp; <span style="font-weight: 700;">Highlights</span>
                    <br>- UHPLC-HRMS identified 13 metabolites predicting IMV and 8 associated to mortality.<br>- FTIR data permitted classification of patients by IMV and death outcomes.<br>- FTIR data excelled UHPLC-HRMS in predictive models with unbalanced groups.<br>-Blood serum metabolomics as a promising tool for predicting ICU patients' prognosis.<br>
                  </p>
                  <a href="https://doi.org/10.1016/j.compbiomed.2024.109393" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
                  <p class="u-small-text u-text u-text-default u-text-variant u-text-3"> Fonseca
TAF, Von Rekowski CP, Araújo R, Oliveira MC, <span style="font-weight: 700;">Justino GC</span>, Bento L, Calado CRC. Comparison
of two metabolomics platforms to discover biomarkers in critically ill patients
from serum analysis. <i>Comp in Biol and Med&nbsp;</i><b>2025</b>, 184, 109393.<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-container-align-center u-section-3" id="sec-e9d1">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="data-layout-selected u-clearfix u-expanded-width u-gutter-0 u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-align-left u-container-align-left u-container-style u-layout-cell u-left-cell u-size-36 u-size-xs-60 u-white u-layout-cell-1" src="">
                <div class="u-container-layout u-container-layout-1">
                  <h3 class="u-text u-text-1"> The
Elusive Biological Activity of Scorpionates: A Useful Scaffold for Cancer
Therapy?&nbsp;</h3>
                  <p class="u-text u-text-2"> Cancer remains a formidable challenge, 
requiring the constant pursuit of novel therapeutic agents and 
strategies. Scorpionates, known for their unique coordination 
properties, have recently gained attention for their anticancer 
potential. Traditionally applied in catalysis, these compounds have 
demonstrated notable cytotoxicity across various cancer cell lines, 
often surpassing the efficacy of conventional chemotherapeutics. This 
review addresses recent findings on scorpionate complexes, emphasizing 
the impact of metal choice and ligand design on biological activity. 
Copper and ruthenium scorpionates show promise, leveraging redox 
activity and mitochondrial disruption mechanisms to selectively induce 
cancer cell death. Ligand modifications, including sulfur-containing 
heterocycles and unsubstituted pyrazoles, have proven effective in 
enhancing cytotoxicity and selectivity. Furthermore, dipodal ligands 
show unique potential, with selective binding sites that improve 
stability and facilitate specific cellular interactions, such as 
targeting metastatic pathways. These findings highlight the largely 
unexplored potential of scorpionate complexes, positioning them as 
candidates for next-generation anticancer therapies. Continued research 
into structure–activity relationships and precise mechanisms of action 
could pave the way for developing highly potent and selective anticancer
 agents based on scorpionate chemistry.<br>
                  </p>
                  <a href="https://www.mdpi.com/1420-3049/29/23/5672" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
                  <p class="u-small-text u-text u-text-default u-text-variant u-text-3"> Petrosian
A, Pinheiro PF, Ribeiro APC, Martins LMDRS, <span style="font-weight: 700;">Justino GC</span>. The Elusive Biological
Activity of Scorpionates: A Useful Scaffold for Cancer Therapy? <i>Molecules</i>,
2024, 29(23), 5672.<br>
                  </p>
                </div>
              </div>
              <div class="u-align-center u-container-align-center u-container-style u-layout-cell u-right-cell u-size-24 u-size-xs-60 u-layout-cell-2" src="">
                <div class="u-container-layout u-container-layout-2" src="">
                  <img class="custom-expanded u-image u-image-1" src="images/vf_GA2.png" data-image-width="1359" data-image-height="1200">
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="sec-1bf2">
      <div class="u-clearfix u-sheet u-sheet-1">
        <p class="u-large-text u-text u-text-variant u-text-1"> Sun Y,
Madureira J, <b>Justino GC</b>, Cabo-Verde S, Chmielewska-Śmietanko D, Sudlitz
M, Bulka S, Chajduk E, Mróz A, Wang S, Wang J. Diclofenac Degradation in
Aqueous Solution Using Electron Beam Irradiation and Combined with
Nanobubbling.<i>
            <br>Appl Sci&nbsp;</i><b>2024</b>, 14, 6028.
        </p>
        <a href="https://doi.org/10.3390/app14146028" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
      </div>
    </section>
    <section class="u-clearfix u-valign-top u-section-5" id="carousel_c5ed">
      <div class="custom-expanded data-layout-selected u-clearfix u-gutter-46 u-layout-wrap u-layout-wrap-1">
        <div class="u-layout" style="">
          <div class="u-layout-col" style="">
            <div class="u-size-30" style="">
              <div class="u-layout-row" style="">
                <div class="u-container-align-center u-container-style u-layout-cell u-size-19 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <img class="custom-expanded u-image u-image-1" src="images/molecules-28-05451-g001-550.jpg" data-image-width="550" data-image-height="341">
                  </div>
                </div>
                <div class="u-container-style u-layout-cell u-size-41 u-layout-cell-2">
                  <div class="u-container-layout u-container-layout-2">
                    <h3 class="u-text u-text-default u-text-1"> Exploring
the Mechanisms behind the Anti-Tumoral Effects of Model C-Scorpionate Complexes</h3>
                    <p class="u-text u-text-default u-text-2"> The growing worldwide cancer incidence, 
coupled to the increasing occurrence of multidrug cancer resistance, 
requires a continuous effort towards the identification of new leads for
 cancer management. In this work, two C-scorpionate complexes were studied as potential scaffolds
 for future anticancer drug development. Their cytotoxicity and cell 
migration inhibitory activity were analyzed, and an untargeted 
metabolomics approach was employed to elucidate the biological processes
 significantly affected by these two complexes, using two tumoral cell 
lines (B16 and HCT116) and a non-tumoral cell line (HaCaT). While the Fe complex did not display a significant cytotoxicity, the Co complex was particularly cytotoxic against the HCT116 cell line. While the Co complex significantly inhibited cell migration in all tested cell lines, the Fe complex displayed a mixed activity. From a metabolomics perspective, exposure to [the Fe complex
 was associated with changes in various metabolic pathways involving 
tyrosine, where iron-dependent enzymes are particularly relevant. On the
 other hand, ​the Co complex
 was associated with dysregulation of cell adhesion and membrane 
structural pathways, suggesting that its antiproliferative and 
anti-migration properties could be due to changes in the overall 
cellular adhesion mechanisms.&nbsp; &nbsp; &nbsp;<br>
                    </p>
                    <a href="https://doi.org/10.3390/molecules28145451" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
                    <p class="u-small-text u-text u-text-default u-text-variant u-text-3"> Silva
PMG, Pinheiro PF, Camões SP, Ribeiro APC, Martins LMDRS, Miranda JPG, <b>Justino
GC*</b>. Exploring the Mechanisms behind the Anti-Tumoral Effects of Model
C-Scorpionate Complexes. <i style="">Molecules&nbsp;</i><b>2023</b>, 28, 5451.
                    </p>
                  </div>
                </div>
              </div>
            </div>
            <div class="u-size-30">
              <div class="u-layout-row">
                <div class="u-container-style u-layout-cell u-size-32 u-layout-cell-3">
                  <div class="u-container-layout u-container-layout-3">
                    <h3 class="u-text u-text-default u-text-4"> An
Optimised MS-Based Versatile Untargeted Metabolomics Protocol</h3>
                    <p class="u-text u-text-default u-text-5"> Untargeted metabolomics approaches require 
complex samples containing the endogenous metabolites of a biological 
system. Here, we describe a set of protocols that can be applied to 
various types of samples, including prokaryotic and eukaryotic cells, as
 well as animal and human samples. Following a single extraction step, 
samples are analysed using different chromatographic conditions coupled 
to high-resolution mass spectrometry. Quantification of metabolite 
changes between samples is performed without internal standards, using 
peak areas from extracted ion chromatograms for statistical analysis. 
Bioinformatics annotation of the results allows a pathway- and 
process-oriented analysis across biological sample conditions, allowing a
 complete pathway interrogation. <br>
                    </p>
                    <a href="https://doi.org/10.3390/separations10050314" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-2" target="_blank">Learn More </a>
                    <p class="u-small-text u-text u-text-default u-text-variant u-text-6"> Marques
CF, <b>Justino GC</b>. An Optimised MS-Based Versatile Untargeted Metabolomics
Protocol. <i>Separations&nbsp;</i><b>2023</b>, 10, 314.
                    </p>
                  </div>
                </div>
                <div class="u-container-style u-layout-cell u-size-28 u-layout-cell-4">
                  <div class="u-container-layout u-container-layout-4">
                    <img class="custom-expanded u-image u-image-contain u-image-2" src="images/separations-10-00314-ag.png" data-image-width="1912" data-image-height="1171">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_80e4">
      <div class="custom-expanded data-layout-selected u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-col">
            <div class="u-size-20">
              <div class="u-layout-row">
                <div class="u-container-style u-layout-cell u-shape-rectangle u-size-19 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <img class="u-image u-image-contain u-image-default u-image-1" src="images/images_large_ao3c01370_0005.jpeg" alt="" data-image-width="879" data-image-height="455">
                  </div>
                </div>
                <div class="u-container-style u-layout-cell u-shape-rectangle u-size-41 u-layout-cell-2">
                  <div class="u-container-layout u-container-layout-2">
                    <h3 class="u-text u-text-default u-text-1"> The Impact of the Serum Extraction Protocol on Metabolomic Profiling Using UPLC-MS/MS and FTIR Spectroscopy<br>
                    </h3>
                    <p class="u-text u-text-2"> Biofluid metabolomics is a very 
appealing tool to increase the knowledge associated with 
pathophysiological mechanisms leading to better and new therapies and 
biomarkers for disease diagnosis and prognosis. However, due to the 
complex process of metabolome analysis, including the metabolome 
isolation method and the platform used to analyze it, there are diverse 
factors that affect metabolomics output. In the present work, the impact
 of two protocols to extract the serum metabolome, one using methanol 
and another using a mixture of methanol, acetonitrile, and water, was 
evaluated. The metabolome was analyzed by ultraperformance liquid 
chromatography associated with tandem mass spectrometry (UPLC-MS/MS), 
based on reverse-phase and hydrophobic chromatographic separations, and 
Fourier transform infrared (FTIR) spectroscopy. The two extraction 
protocols of the metabolome were compared over the analytical platforms 
(UPLC-MS/MS and FTIR spectroscopy) concerning the number of features, 
the type of features, common features, and the reproducibility of 
extraction replicas and analytical replicas. The ability of the 
extraction protocols to predict the survivability of critically ill 
patients hospitalized at an intensive care unit was also evaluated. The 
FTIR spectroscopy platform was compared to the UPLC-MS/MS platform and, 
despite not identifying metabolites and consequently not contributing as
 much as UPLC-MS/MS in terms of information concerning metabolic 
information, it enabled the comparison of the two extraction protocols 
as well as the development of very good predictive models of patient’s 
survivability, such as the UPLC-MS/MS platform. Furthermore, FTIR 
spectroscopy is based on much simpler procedures and is rapid, economic,
 and applicable in the high-throughput mode, i.e., enabling the 
simultaneous analysis of hundreds of samples in the microliter range in a
 couple of hours. Therefore, FTIR spectroscopy represents a very 
interesting complementary technique not only to optimize processes as 
the metabolome isolation but also for obtaining biomarkers such as those
 for disease prognosis.<br>
                    </p>
                    <a href="https://doi.org/10.1021/acsomega.3c01370" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
                    <p class="u-small-text u-text u-text-default u-text-variant u-text-3"> Fonseca TAH, Von Rekowski CP, Araújo R, Oliveira
MC, <b>Justino GC</b>, Bento L, Calado CRC. The Impact
of the Serum Extraction Protocol on Metabolomic Profiling Using UPLCMS/ MS and
FTIR Spectroscopy. <i>ACS Omega</i>. <b>2023</b> 8:20755-20766.
                    </p>
                  </div>
                </div>
              </div>
            </div>
            <div class="u-size-20">
              <div class="u-layout-row">
                <div class="u-container-style u-layout-cell u-shape-rectangle u-size-39 u-layout-cell-3">
                  <div class="u-container-layout u-container-layout-3">
                    <h3 class="u-text u-text-4"> Protocol to study in&nbsp;vitro drug metabolism and identify montelukast metabolites from purified enzymes and primary cell cultures by mass spectrometry</h3>
                    <p class="u-text u-text-default u-text-5">
                      <span style="font-weight: 700;">Highlights</span>
                      <br>- An in&nbsp;vitro set of assays that can be used in drug metabolism studies<br>- A high-resolution electrospray ionization mass spectrometry analytical tool<br>- Characterization of drug metabolites by mass spectrometry<br>- Small-scale metabolite identification<br>
                    </p>
                    <a href="https://doi.org/10.1016/j.xpro.2023.102086" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-2" target="_blank">Learn More </a>
                    <p class="u-small-text u-text u-text-default u-text-variant u-text-6"> Marques CF, Pinheiro PF, <b>Justino GC</b>. Protocol to study in vitro drug metabolism and identify montelukast
metabolites from purified enzymes and primary cell cultures by mass
spectrometry. <i>STAR Protoc</i>. <b>2023</b> 4(1):102086.
                    </p>
                  </div>
                </div>
                <div class="u-container-style u-layout-cell u-shape-rectangle u-size-21 u-layout-cell-4">
                  <div class="u-container-layout u-container-layout-4">
                    <img class="custom-expanded u-image u-image-contain u-image-2" src="images/1-s2.0-S2666166723000448-fx1.jpg" data-image-width="375" data-image-height="375">
                  </div>
                </div>
              </div>
            </div>
            <div class="u-size-20">
              <div class="u-layout-row">
                <div class="u-container-style u-layout-cell u-size-23 u-layout-cell-5">
                  <div class="u-container-layout u-container-layout-5">
                    <img class="u-image u-image-contain u-image-default u-image-3" src="images/cancers-15-00270-ag.png" alt="" data-image-width="1100" data-image-height="1395">
                  </div>
                </div>
                <div class="u-container-style u-layout-cell u-shape-rectangle u-size-37 u-layout-cell-6">
                  <div class="u-container-layout u-container-layout-6">
                    <h3 class="u-text u-text-7"> Macrophage
Resistance to Ionizing Radiation Exposure Is Accompanied by Decreased Cathepsin
D and Increased Transferrin Receptor 1 Expression</h3>
                    <p class="u-text u-text-default u-text-8"> Resistance to treatment, particularly to 
radiotherapy, is still a major clinical problem in cancer management. 
Macrophages are abundant immune cells at the tumor microenvironment, 
being exposed to ionizing radiation during cancer radiotherapy. 
Considering the role of macrophages in tumor progression and therapy 
outcome, it is crucial to investigate their response to clinically 
relevant ionizing radiation doses for the design of new strategies to 
overcome tumor radio resistance. In this work, we have used a proteomic 
approach to evaluate the expression profile of irradiated versus 
non-irradiated macrophages. This analysis, supported by validation using
 cell-based assays, led to the identification of two main deregulated 
targets, cathepsin D and transferrin receptor 1, in irradiated 
macrophages. Investigating macrophage response to ionizing radiation 
could lead to the identification of deregulated pathways and molecular 
players that can be targeted to overcome tumor radio resistance.<br>
                    </p>
                    <a href="https://doi.org/10.3390/cancers15010270" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-3" target="_blank">Learn More </a>
                    <p class="u-small-text u-text u-text-default u-text-variant u-text-9"> Pinto AT*, Machado AB, Osório H, Pinto ML,
Vitorino R, <b>Justino GC</b>, Santa C, Castro F, Cruz T, Lima J, Cardoso AP,
Figueira R, Monteiro A, Marques M, Manadas B, Pauwels J, Gevaert K, Mareel M,
Rocha S, Duarte T, Oliveira, MJ. Macrophage Resistance
to Ionizing Radiation Exposure Is Accompanied by Decreased Cathepsin D and
Increased Transferrin Receptor 1 Expression. <i>Cancers</i><b>2023</b> 15(1):270.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-7" id="carousel_9cc7">
      <div class="u-clearfix u-sheet u-sheet-1">
        <p class="u-large-text u-text u-text-variant u-text-1"> RN Bento, MA Rendas, VAR Semedo, CF Marques, GC
Justino, CES Bernardes, MEM Piedade, F Antunes. Energy
dissipation in early detection of cellular responses to metabolic challenges. <i>bioRxiv</i><b>2022</b>
        </p>
        <a href="https://www.biorxiv.org/content/10.1101/2022.08.03.502646v1" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
      </div>
    </section>
    <section class="u-clearfix u-section-8" id="sec-5074">
      <div class="u-clearfix u-sheet u-valign-bottom u-sheet-1">
        <div class="data-layout-selected u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-col">
              <div class="u-size-30">
                <div class="u-layout-row">
                  <div class="u-container-style u-layout-cell u-size-30 u-layout-cell-1">
                    <div class="u-container-layout u-container-layout-1">
                      <img class="u-expanded-width u-image u-image-contain u-image-default u-image-1" src="images/1-s2.0-S0024320522007561-ga1_lrg.jpg" alt="" data-image-width="2213" data-image-height="776">
                    </div>
                  </div>
                  <div class="u-container-style u-layout-cell u-size-30 u-layout-cell-2">
                    <div class="u-container-layout u-container-layout-2">
                      <h3 class="u-text u-text-default u-text-1"> The
mechanisms underlying montelukast's neuropsychiatric effects – new insights
from a combined metabolic and multiomics approach</h3>
                      <p class="u-text u-text-default u-text-2">Montelukast (MTK) is an antagonist of the <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cysteinyl-leukotriene-receptor-1" title="Learn more about cysteinyl leukotrienes receptor 1 from ScienceDirect's AI-generated Topic Pages" class="topic-link u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">cysteinyl leukotrienes receptor 1</a> widely used to manage asthma symptoms among adults and children. 
However, it has been associated with an increasing number of 
neuropsychiatric <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/adverse-drug-reaction" title="Learn more about adverse drug reactions from ScienceDirect's AI-generated Topic Pages" class="topic-link u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">adverse drug reactions</a> (ADRs), particularly among children, including depression, sleep 
disturbance, and suicidal ideation. The aims of this work were to 
characterize MTK metabolism in vitro and in vivo and to identify its effects at the <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/metabolome" title="Learn more about metabolome from ScienceDirect's AI-generated Topic Pages" class="topic-link u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">metabolome</a> and <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/proteome" title="Learn more about proteome from ScienceDirect's AI-generated Topic Pages" class="topic-link u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">proteome</a> levels in order to explain its toxicity. Results
 clearly indicate that montelukast therapeutic effects are accompanied 
by a strong modulation of specific processes in the central nervous 
system that may explain the observed neuropsychiatric reactions. 
Moreover, the results also suggest that adverse drug reactions are more 
likely to occur in children, due to the early maturation stage of their 
brains.<br>
                      </p>
                      <a href="https://doi.org/10.1016/j.lfs.2022.121056" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-5" target="_blank">Learn More </a>
                      <p class="u-small-text u-text u-text-default u-text-variant u-text-3"> Marques CF, Marques MM, <b>Justino GC</b>. The mechanisms underlying montelukast's neuropsychiatric effects –
new insights from a combined metabolic and multiomics approach. <i>Life Sci</i>. <b>2022</b> 310:121056.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
              <div class="u-size-30">
                <div class="u-layout-row">
                  <div class="u-container-style u-layout-cell u-size-39 u-layout-cell-3">
                    <div class="u-container-layout u-container-layout-3">
                      <h3 class="u-text u-text-4"> Optimized
protocol for obtaining and characterizing primary neuron-enriched cultures from
embryonic chicken brains</h3>
                      <p class="u-text u-text-default u-text-5"> Highlights<br>An in&nbsp;vitro model that can be used for Alzheimer disease studies<br>Fast and accessible protocol with no need for an animal facility<br>Isolation of a neuron-enriched cell fraction from chicken embryos<br>Characterization of the neuronal model using proteomics and cell staining approaches<br>
                      </p>
                      <a href="https://doi.org/10.1016/j.xpro.2022.101753" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-6" target="_blank">Learn More </a>
                      <p class="u-small-text u-text u-text-default u-text-variant u-text-6"> Marques CF, Pinheiro PF, <b>Justino GC</b>. Optimized protocol for obtaining and characterizing primary
neuron-enriched cultures from embryonic chicken brains. <i>STAR Protoc</i>. <b>2022</b> 3(4):101753.<br>
                      </p>
                    </div>
                  </div>
                  <div class="u-container-style u-layout-cell u-size-21 u-layout-cell-4">
                    <div class="u-container-layout u-valign-top u-container-layout-4">
                      <img class="u-image u-image-default u-preserve-proportions u-image-2" src="images/1-s2.0-S2666166722006335-fx1_lrg.jpg" alt="" data-image-width="996" data-image-height="996">
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-9" id="sec-d271">
      <div class="u-clearfix u-sheet u-sheet-1">
        <p class="u-large-text u-text u-text-variant u-text-1"> Marques CF, Marques MM, <b>Justino GC</b>. Leukotrienes vs. Montelukast-Activity, Metabolism, and Toxicity
Hints for Repurposing. <i>Pharmaceuticals (Basel)</i>. <b>2022</b> 15(9):1039.&nbsp;
        </p>
        <a href="https://doi.org/10.3390/ph15091039" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
      </div>
    </section>
    <section class="u-clearfix u-container-align-center u-section-10" id="sec-ce34">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="data-layout-selected u-clearfix u-expanded-width u-gutter-0 u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-align-left u-container-align-left u-container-style u-layout-cell u-left-cell u-size-36 u-size-xs-60 u-white u-layout-cell-1" src="">
                <div class="u-container-layout u-container-layout-1">
                  <h3 class="u-text u-text-1"> Molecular dynamics simulations and analysis for bioinformatics undergraduate students</h3>
                  <p class="u-text u-text-2"> A computational biochemistry laboratory, fitted for 
bioinformatics students, is presented. The molecular dynamics package 
GROMACS is used to prepare and simulate a solvated protein. Students 
analyze the trajectory with different available tools (GROMACS and VMD) 
to probe the structural stability of the protein during the simulation. 
Students are also required to make use of Python libraries and write 
their own code to probe non-covalent interactions between the amino acid
 side chains. Based on these results, students characterize the system 
in a qualitatively approach but also assess the importance of each 
specific interaction through time. This work mobilizes biochemical 
concepts and programming skills, fostering critical thinking and group 
work and developing presenting skills.&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>
                  </p>
                  <a href="https://doi.org/10.1002/bmb.21512" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
                  <p class="u-small-text u-text u-text-default u-text-variant u-text-3"><b>Justino GC</b>, Nascimento CP, Justino MC. Molecular
dynamics simulations and analysis for bioinformatics undergraduate students. <i>Biochem
Mol Biol Educ.</i><b>2021</b> 49(4):570-582.<br>
                  </p>
                </div>
              </div>
              <div class="u-align-center u-container-align-center u-container-style u-image u-layout-cell u-right-cell u-size-24 u-size-xs-60 u-image-1" src="" data-image-width="2128" data-image-height="681">
                <div class="u-container-layout u-container-layout-2" src=""></div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-container-align-center u-section-11" id="sec-2df0">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="data-layout-selected u-clearfix u-expanded-width u-gutter-0 u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-align-center u-container-align-center u-container-style u-image u-layout-cell u-right-cell u-size-24 u-size-xs-60 u-image-1" src="" data-image-width="713" data-image-height="651">
                <div class="u-container-layout u-container-layout-1" src=""></div>
              </div>
              <div class="u-align-left u-container-align-left u-container-style u-layout-cell u-left-cell u-size-36 u-size-xs-60 u-white u-layout-cell-2" src="">
                <div class="u-container-layout u-valign-top u-container-layout-2">
                  <h3 class="u-text u-text-1"> Immobilization
of His-tagged proteins on NiO foams for recyclable enzymatic reactors</h3>
                  <p class="u-text u-text-2"> Highlights<br>NiO foams with high <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/surface-area" title="Learn more about surface area from ScienceDirect's AI-generated Topic Pages" class="topic-link u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">surface area</a> were synthesised.<br>His-tag proteins bind to the NiO foam and remain bound upon wash.<br>
                    <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/protein-binding" title="Learn more about Protein binding from ScienceDirect's AI-generated Topic Pages" class="topic-link u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">Protein binding</a> to the foam results in <a href="https://www.sciencedirect.com/topics/chemistry/micro-porosity" title="Learn more about microporous from ScienceDirect's AI-generated Topic Pages" class="topic-link u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">microporous</a> cavities alterations.<br>
                    <a href="https://www.sciencedirect.com/topics/earth-and-planetary-sciences/density-functional-theory" title="Learn more about DFT from ScienceDirect's AI-generated Topic Pages" class="topic-link u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">DFT</a> suggests His-tags prefer stretched conformations and act as bidentate ligands.<br>Immobilized His-tag enzyme remains active and the support is recyclable.<br>
                  </p>
                  <a href="https://doi.org/10.1016/j.apsusc.2020.147848" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-5" target="_blank">Learn More </a>
                  <p class="u-small-text u-text u-text-default u-text-variant u-text-3"> Rosado PR,
Meyrelles R, Macatrão AM, Justino MC, Gomes AG, Montemor MF, Alves MM, <b>Justino
GC,</b> Ribeiro APC, Shimizu K. Immobilization of His-tagged proteins on NiO
foams for recyclable enzymatic reactors <i>Appl Surf Sci</i><b>2021</b> 537:147848.<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-container-align-center u-section-12" id="sec-29b1">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="data-layout-selected u-clearfix u-expanded-width u-gutter-0 u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-align-left u-container-align-left u-container-style u-layout-cell u-left-cell u-size-36 u-size-xs-60 u-white u-layout-cell-1" src="">
                <div class="u-container-layout u-container-layout-1">
                  <h3 class="u-text u-text-1"> Flexible ZnO-mAb
nanoplatforms for selective peripheral blood mononuclear cell immobilization.</h3>
                  <p class="u-text u-text-2"> Cancer is 
the second cause of death worldwide. This devastating disease requires 
specific, fast, and affordable solutions to mitigate and reverse this 
trend. A step towards cancer-fighting lies in the isolation of natural 
killer (NK) cells, a set of innate immune cells, that can either be used
 as biomarkers of tumorigenesis or, after autologous transplantation, to
 fight aggressive metastatic cells. In order to specifically isolate NK 
cells (which express the surface NKp30 receptor) from peripheral blood 
mononuclear cells, a ZnO immunoaffinity-based platform was developed by 
electrodeposition of the metal oxide on a flexible indium tin oxide 
(ITO)-coated polyethylene terephthalate (PET) substrate. The resulting 
crystalline and well-aligned ZnO nanorods (NRs) proved their efficiency 
in immobilizing monoclonal anti-human NKp30 antibodies (mAb), obviating 
the need for additional procedures for mAb immobilization. The presence 
of NK cells on the peripheral blood mononuclear cell (PBMCs) fraction 
was evaluated by the response to their natural ligand (B7-H6) using an 
acridine orange (AO)-based assay. The successful selection of NK cells 
from PBMCs by our nanoplatform was assessed by the photoluminescent 
properties of AO. This easy and straightforward ZnO-mAb nanoplatform 
paves the way for the design of biosensors for clinic diagnosis, and, 
due to its inherent biocompatibility, for the initial selection of NK 
cells for autotransplantation immunotherapies.<br>
                  </p>
                  <a href="https://doi.org/10.1038/s41598-020-72133-0" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
                  <p class="u-small-text u-text u-text-default u-text-variant u-text-3"> Babu KS, Pinheiro PF, Marques CF, <b>Justino GC,</b> Andrade SM, Alves MM.
Flexible ZnO-mAb nanoplatforms for selective peripheral blood
mononuclear cell immobilization. <i>Sci Rep.</i><b>2020</b> 10(1):15018<br>
                  </p>
                </div>
              </div>
              <div class="u-align-center u-container-align-center u-container-style u-image u-layout-cell u-right-cell u-size-24 u-size-xs-60 u-image-1" src="" data-image-width="1494" data-image-height="1181">
                <div class="u-container-layout u-container-layout-2" src=""></div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-container-align-center u-section-13" id="sec-110b">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="data-layout-selected u-clearfix u-expanded-width u-gutter-0 u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-align-center u-container-align-center u-container-style u-image u-layout-cell u-right-cell u-size-24 u-size-xs-60 u-image-1" src="" data-image-width="2128" data-image-height="2887">
                <div class="u-container-layout u-container-layout-1" src=""></div>
              </div>
              <div class="u-align-left u-container-align-left u-container-style u-layout-cell u-left-cell u-size-36 u-size-xs-60 u-white u-layout-cell-2" src="">
                <div class="u-container-layout u-container-layout-2">
                  <h3 class="u-text u-text-1"> NKp30 - A prospective
target for new cancer immunotherapy strategies.&nbsp;</h3>
                  <p class="u-text u-text-2"> Natural killer (NK) cells are an important arm of the 
innate immune system. They constitutively express the NKp30 receptor. 
NKp30-mediated responses are triggered by the binding of specific 
ligands e.g. tumour cell-derived B7-H6 and involve the secretion of 
cytotoxic mediators including TNF-α, IFN-γ, perforins and granzymes. The
 latter two constitute a target cell-directed response that is critical 
in the process of immunosurveillance. The structure of NKp30 is 
presented, focusing on the ligand-binding site, on the ligand-induced 
structural changes and on the experimental data available correlating 
structure and binding affinity. The translation of NKp30 structural 
changes to disease progression is also reviewed. NKp30 role in 
immunotherapy has been explored in chimeric antigen receptor T-cell 
(CAR-T) therapy. However, antibodies or small ligands targeting NKp30 
have not yet been developed. The data reviewed herein unveil the key 
structural aspects that must be considered for drug design in order to 
develop novel immunotherapy approaches. <br>
                  </p>
                  <a href="https://doi.org/10.1111/bph.15222" class="u-border-2 u-border-palette-2-base u-btn u-btn-round u-button-style u-hover-palette-2-base u-none u-radius u-text-body-color u-text-hover-white u-btn-1" target="_blank">Learn More </a>
                  <p class="u-small-text u-text u-text-default u-text-variant u-text-3"> Pinheiro PF, <b>Justino GC</b>, Marques MM. NKp30 - A
prospective target for new cancer immunotherapy strategies. <i>Br J Pharmacol.</i><b>2020</b> 177(20):4563-4580.<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-14" id="sec-9539">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <p class="u-align-center u-large-text u-text u-text-variant u-text-1">
          <a class="u-active-none u-border-none u-btn u-button-link u-button-style u-hover-none u-none u-text-palette-2-base u-btn-1" href="https://orcid.org/0000-0003-4828-4738">Full Publication List</a>
        </p>
      </div>
    </section>
    <section class="u-align-center u-clearfix u-container-align-center u-image u-shading u-section-15" data-image-width="1980" data-image-height="806" id="sec-4c1c">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="u-social-icons u-spacing-20 u-social-icons-1">
          <a class="u-social-url" target="_blank" href="https://www.instagram.com/chembiolab_cqe/" title="ChemBioLab"><span class="u-icon u-icon-circle u-social-icon u-social-instagram u-text-white u-icon-1"><svg class="u-svg-link" preserveAspectRatio="xMidYMin slice" viewBox="0 0 112 112" style=""><use xlink:href="#svg-a970"></use></svg><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 112 112" x="0px" y="0px" id="svg-a970"><path d="M55.9,32.9c-12.8,0-23.2,10.4-23.2,23.2s10.4,23.2,23.2,23.2s23.2-10.4,23.2-23.2S68.7,32.9,55.9,32.9z M55.9,69.4c-7.4,0-13.3-6-13.3-13.3c-0.1-7.4,6-13.3,13.3-13.3s13.3,6,13.3,13.3C69.3,63.5,63.3,69.4,55.9,69.4z"></path><path d="M79.7,26.8c-3,0-5.4,2.5-5.4,5.4s2.5,5.4,5.4,5.4c3,0,5.4-2.5,5.4-5.4S82.7,26.8,79.7,26.8z"></path><path d="M78.2,11H33.5C21,11,10.8,21.3,10.8,33.7v44.7c0,12.6,10.2,22.8,22.7,22.8h44.7c12.6,0,22.7-10.2,22.7-22.7 V33.7C100.8,21.1,90.6,11,78.2,11z M91,78.4c0,7.1-5.8,12.8-12.8,12.8H33.5c-7.1,0-12.8-5.8-12.8-12.8V33.7 c0-7.1,5.8-12.8,12.8-12.8h44.7c7.1,0,12.8,5.8,12.8,12.8V78.4z"></path></svg>
          
          
        </span>
          </a>
          <a class="u-social-url" target="_blank" data-type="Email" title="Email" href="mailto:goncalo.justino@estbarreiro.ips.pt"><span class="u-file-icon u-icon u-social-email u-social-icon u-text-white u-icon-2"><img src="images/3892862-c006f9e1.png" alt=""></span>
          </a>
          <a class="u-social-url" target="_blank" data-type="Dribbble" title="ORCID" href="https://orcid.org/0000-0003-4828-4738"><span class="u-file-icon u-icon u-social-dribbble u-social-icon u-icon-3"><img src="images/ORCID-iD_icon_32x32.png" alt=""></span>
          </a>
        </div>
      </div>
    </section>
    
    
    
    
    <section class="u-backlink u-clearfix u-grey-80">
      <p class="u-text">
        <span>This site was created with the </span>
        <a class="u-link" href="https://nicepage.com/" target="_blank" rel="nofollow">
          <span>Nicepage</span>
        </a>
      </p>
    </section>
  
</body></html>